packetDeserializer.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+// deserializePacket Will contain parameters to index on elasticsearch
+type deserializedPacket struct {
```

Comment:
#5 was opened to discuss these parameters. We still don't have a clear idea on this.
Maybe checking moloch's demo is a good idea or maybe start with some basic info to index for the sake of MVP.

Reply:
A related thought here, Moloch displays sessions in its Sessions page
![image](https://user-images.githubusercontent.com/24691797/70651131-4eae8400-1c76-11ea-9057-76357a9b3b1c.png)
Each row in the above screen grab is a session. While we are decoding packets and getting info about each packet, I personally think that it makes more sense to first get Flow data.
Each row in our case can be a Flow(having attributes such as src, dest, num_of_packets etc.), which on further expansion can yield info about packets consisting the particular Flow.
While packet info can be a great metric in case of aggregated visualisations, it does not make sense for any end user to come looking for a specific packet and inspect packet info. Instead it makes much more sense to have a Flow first, with packets grouped according to Endpoints or FlowHash. 

Deserialized packet data will ofcourse be of great use case for visualisations but as a project intended for users coming to get an idea of their network, Flows make more sense IMO.

This should have been among our first points of discussion earlier. Let me know your take on this, and we can have a separate issue thread discussing the same.
Until then let's put this PR on hold.